tests/Doctrine/Tests/DBAL/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -1360,4 +1361,26 @@ public function commentsProvider() : array
             'valid with extra opening brackets' => ['(DC2Type:should((.stop)).before)', 'should((.stop', $currentType],
         ];
     }
+
+    public function testCreateAndListSequences() : void
+    {
+        if ( ! $this->_sm->getDatabasePlatform()->supportsSequences()) {
+            $this->markTestSkipped('This test is only supported on platforms that support sequences.');
+        }
+
+        $sequence1Name = 'sequence_1';
+        $sequence2Name = 'sequence_2';
+        $sequence1     = new Sequence($sequence1Name);
```

Comment:
@simPod please make sure all sequence attributes are covered by the test, not only the name.

Reply:
Can you review now please? Thanks